client/galaxy/scripts/mvc/tool/tool-form.js

```diff
@@ -68,15 +68,15 @@ var View = Backbone.View.extend({
                                     );
                                 } else {
                                     Galaxy.modal &&
```

Comment:
@guerler Do you remember why we needed to have this guard in here?  Is it irrelevant now, and safe to drop?

Reply:
Yes, it should be safe to remove that safe guard now.